CppImport/src/ClangHelpers.cpp

```diff
@@ -29,6 +29,9 @@
 #include "OOModel/src/declarations/Project.h"
 #include "OOModel/src/declarations/MetaDefinition.h"
 
+#include "CppExport/src/exporter/CppExporter.h"
```

Comment:
When adding dependencies, please modify the `CMakeLists.txt`, `precompiled.h` and the `.plugin` files to add the corresponding dependency there.